riscv-overlay-software-standard-draft.adoc

```diff
@@ -52,16 +52,17 @@ link:#instrumentation-interface-hooks[Table 1 - Instrumentation interface hooks]
 [[_Toc507430300]]**Abbreviations**
 
 [cols=",",options="",]
-|===========================
-|Abbreviation |Description
-|SW |Software
-|HW |Hardware
-|MMU |Memory Management Unit
-|OS |Operation System
-|LRU |Least Recently Used
-|RT |RunTime or RealTime
-|R/O or RO |Read Only
-| |
++|===========================
++|Abbreviation |Description
++|SW           |Software
++|HW           |Hardware
++|MMU          |Memory Management Unit
++|LRU          |Least Recently Used
++|OS           |Operation System
++|RT           |RunTime or RealTime
```

Comment:
Runtime and realtime mean two different things, so we can't use the same acronym for both.

Reply:
I agree, but sometimes people don't catch the difference. 
I will fix it